test/test_events.py

```diff
@@ -179,10 +179,12 @@ def test_seminar_events_typical(uqcsbot: MockUQCSBot):
     messages = uqcsbot.test_messages.get(TEST_CHANNEL_ID, [])
     assert len(messages) == 2
     expected = ("Events in the *next _2_ weeks*\r\n"
-                "*MAY 28 12:00 - 13:00* - `<https://www.itee.uq.edu.au/introduction"
+                "*TUE MAY 28 12:00 - 13:00* - "
+                "`<https://www.itee.uq.edu.au/introduction"
                 + "-functional-programming|Introduction to functional programming"
                 + " - Tony Morris, Software Engineer at Data61>` - _78-420_\r\n"
-                "*MAY 29 13:00 - 14:00* - `<https://www.itee.uq.edu.au/performance-enhancement-"
+                "*WED MAY 29 13:00 - 14:00* - "
+                  "`<https://www.itee.uq.edu.au/performance-enhancement-"
```

Comment:
Message line continued; begin with `+`.
Move joining characters (` - `) to latter line.
Also, this line is slightly over indented.